### ISSUE TITLE
Домен .ru/.рф

### ISSUE THREAD

**Issue by valericus** [stouza/stouza.ru#1]
Мне кажется не самым удачным решением использовать эти доменные имена, если речь идёт о защите от государственного произвола. Регистраторы в этих зонах очень подвержены влиянию властей, вплоть до произвольного разделегирования (см. случай torrents.ru).

**Comment by dmikushin** [comment]
Согласен, добавлю ещё другие. В принципе, сайт - не главная точка входа. Инструмент донейтов - это в основном мобильное приложение. И для фондов - тоже, когда будет найдено безопасное решение без виртуалки.
